connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointTask.java

```diff
@@ -81,7 +93,14 @@ public void start(Map<String, String> props) {
         adminTimeout = config.adminTimeout();
         offsetSyncStore = new OffsetSyncStore(config);
         sourceAdminClient = AdminClient.create(config.sourceAdminConfig());
+        targetAdminClient = (KafkaAdminClient) Admin.create(config.targetAdminConfig());
         metrics = config.metrics();
+        scheduler = new Scheduler(MirrorCheckpointTask.class, config.adminTimeout());
+        if (config.groupOffsetSyncEnabled()) {
```

Comment:
Nit: the scheduler understands negative intervals as "disabled", so you don't need to have this extra check here. Just have syncGroupOffsetInterval return -1 if it's disabled.

Reply:
removed the `if` condition